package.json

```diff
@@ -20,7 +20,6 @@
     "lint-sass": "stylelint client/src"
   },
   "dependencies": {
-    "@silverstripe/react-injector": "^0.2.1",
```

Comment:
This is an standalone version of react-injector as opposed to the one that's in silverstripe/admin

I honestly don't know why it exists, silverstripe/mfa is the only thing that uses it and silverstripe/admin is a required composer dependency.

Reply:
Update - it exists for when you register mfa before logging into admin and all the admin js isn't availalble.  If you update mfa in inside of admin on the edit user form then it can just use the admin injector at that point

Reply:
We should ideally remove this dep and export out the injector into the `client/dist/js/` dir in admin. We can then include it via the requirements API.

Happy to do that as a separate issue though if you'd prefer that.

Reply:
I'd say separate issue for this one - have backlogged a card https://github.com/silverstripe/silverstripe-admin/issues/1891